packages/api/cms-api/src/auth/strategies/static-authed-user.strategy.ts

```diff
@@ -6,7 +6,7 @@ import { ExtractJwt, Strategy } from "passport-jwt";
 import { CurrentUserInterface } from "../current-user/current-user";
 
 interface StaticAuthedUserStrategyConfig {
-    staticAuthedUser: CurrentUserInterface;
+    staticAuthedUser: CurrentUserInterface | string;
```

Comment:
I don't understand this change: If I pass a string here, is this the `userIdentifier` used in `validate`? In your example, how could the user identifier ever be anything different than `admin`? Especially as the JWT is always signed with `config.staticAuthedUser`?

Reply:
This is the result of staying backwards compatible (passing the user-object already to the factory) and adding the possibility to load the static user in validate (like in https://github.com/vivid-planet/comet/blob/e97c77c141f0f4098a2b7ea4a385491ba1f791cf/demo/api/src/auth/auth.strategy.ts).
It's not perfect but I think it's okay.

Reply:
> This is the result of staying backwards compatible (passing the user-object already to the factory)

There's no need to stay backwards compatible, as this change only affects next and has not been published in a stable version yet. Could we make this more elegant by introducing a breaking change?

Reply:
Is this really still the use-case for StaticAuthedUserStategy?

wouldn't it be more appropriate to create a new AuthStrategy from scratch?

Reply:
I changed the PR and added a strategy.